app/models/spree/user.rb

```diff
@@ -47,6 +48,16 @@ def self.admin_created?
       User.admin.count > 0
     end
 
+    def self.from_omniauth(auth)
+      where(provider: auth.provider, uid: auth.uid).first_or_create do |user|
```

Comment:
We could also consider that most people are probably going to sign up for Les Common specifically in order to authenticate with OFN and other platforms on the DFC. So perhaps this should actually be

```suggestion
      where(email: auth.info.email).first_or_create do |user|
```

Needs some thought and/or input from @pacodelaluna 🙏

Reply:
Where is the authorisation?

My theory is that we verify the provider's message with their public key and if we are sure the message comes from them, we can log in anyone here, just taking the email.